src/types/onyx/OriginalMessage.ts

```diff
@@ -270,20 +270,140 @@ type OriginalMessageChangeLog = {
     /** When was it last modified */
     lastModified?: string;
 
-    /** New role of user */
-    newValue?: string;
+    /** New role of user or new value of the category/tag field
+     * The user role will be of type string and category/tag value (enabled/disable)
+     * will be of type boolean.
+     */
+    newValue?: boolean | string;
 
-    /** Old role of user */
-    oldValue?: string;
+    /** Old role of user or old value (enabled/disable) of the category/tag field.
+     * The user role will be of type string and category/tag value (enabled/disable)
+     * will be of type boolean.
+     */
+    oldValue?: boolean | string;
 
     /** Name of connection */
     connectionName?: AllConnectionName;
 
-    /** Old auto-reporting frequency */
-    oldFrequency?: string;
+    /** Name of the added category */
+    categoryName?: string;
+};
+
+/** Model of change log */
+type OriginalMessagePolicyChangeLog = {
+    /** Account IDs of users that either got invited or removed from the room */
+    targetAccountIDs?: number[];
+
+    /** Name of the chat room */
+    roomName?: string;
+
+    /** Description of the chat room */
+    description?: string;
+
+    /** ID of the report */
+    reportID?: number;
```

Comment:
@rushatgabhane @Krishna2323 is it ok that this is a `number`? wound't this have problems with large reportIDs?